velox/common/base/tests/Filesystem.h

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Should we name this `FilesystemMacro.h`? Or simply put inside `Macros.h`. `Filesystem.h` seems like it probably contains a class definition to me.

Reply:
I was thinking about that, but there is no macro :-)

Reply:
Maybe Fs.h?

Reply:
Yeah, Fs.h seems like a better choice.